Storage/SQL/SQLiteHistory.swift

```diff
@@ -82,18 +82,30 @@ public class SQLiteHistory {
 
     private var ignoredSchemes = ["about"]
 
-    required public init(db: BrowserDB) {
+    required public init?(db: BrowserDB, version: Int? = nil) {
```

Comment:
Hm. I'd believe you, but every call site I see in the tests below does `SQLiteHistory(…)!`

Am I missing something?